code/config.ss

```diff
@@ -0,0 +1,154 @@
+(define track-config
+  '((language . "Scheme")
+    
+    (active . #t)
+
+    (blurb . "")
+
+    (exercises
+     
+     ((slug . hello-world)
+      (uuid . "f2936612-edc4-44f9-a403-c60f7c6486e4")
+      (core . #f)
+      (auto_approve . #t)
+      (unlocked_by)
+      (difficulty . 1)
+      (topics))
+     
+     ((slug . hamming)
+      (uuid . "ce513303-471b-459a-bcf1-6c69f4d83ae7")
+      (core . #f)
+      (unlocked_by)
+      (difficulty . 1)
+      (topics list))
+     
+     ((slug . leap)
+      (uuid . "b94548df-3b35-4ce1-80c7-36179d2b3b86")
+      (core . #f)
+      (unlocked_by)
+      (difficulty . 1)
+      (topics))
+     
+     ((slug . grains)
+      (uuid . "397d9cfc-43f9-4c57-bc55-5c94ae8f532e")
+      (core . #f)
+      (unlocked_by)
+      (difficulty . 1)
+      (topics math))
+     
+     ((slug . bob)
+      (uuid . "4cf6697c-ad9e-426c-a2d8-6c3695812421")
+      (core . #f)
+      (unlocked_by)
```

Comment:
I see my approach to this problem on the PR to your fork is already in line with what you have here.  `Renaissance` must have gotten behind without me realizing.

Should I change that PR to your `schemify` branch, so it can be auto included in this PR once we agree on it over there?